src/Profile/EditProfile.tsx

```diff
@@ -57,72 +58,80 @@ const CreateOrEditProfile = (props: Props) => {
         let passwordLengthError: boolean = false;
         let passwordMatchError: boolean = false;
         let phoneNumberError: boolean = false;
-
+        let alphanumericError: boolean = false;
 
         if (!profile.firstName || profile.firstName === "") {
             nameError = true;
         }
         if (!profile.lastName || profile.lastName === "") {
             lastNameError = true;
         }
-        if (!password || password.length<7) {
+        if (!profile.password || profile.password.length<7) {
             passwordLengthError = true;
         }
-        if (password !== confirmPassword) {
+        if (profile.password !== confirmPassword) {
             passwordMatchError = true;
         }
         if (!profile.phoneNumber || profile.phoneNumber === "") {
             phoneNumberError = true;
         }
+        if (!profile.password || !regexp.test(profile.password)){
+            alphanumericError = true;
+        }
 
         const newErrors: Errors = {
             nameError,
             lastNameError,
             passwordLengthError,
             passwordMatchError,
-            phoneNumberError
+            phoneNumberError,
+            alphanumericError
         }
 
         return newErrors;
     }
 
     return (
-        <div className={"create-book"}>
+        <div >
```

Comment:
Some recommendations:
- Always add classnames to your screens (to the first div)
- On the .css of your screen always select first you screen className to avoid applying the style to other elements in other places. 
For example, you want to change the style of an `<h1>` with the className "title" inside your login screen with className `login-screen`. To do so:

```
.login-screen .title {
    /* change your text styles here */
}
```